### ISSUE TITLE
no wildcard support 

### ISSUE THREAD

**Issue by piyu7274** [google/grumpy#337]
raceback (most recent call last):
  File "tools/grumpc", line 118, in <module>
    sys.exit(main(parser.parse_args()))
  File "tools/grumpc", line 76, in main
    visitor.visit(mod)
  File "/home/avdhesh/grumpy/build/lib/python2.7/site-packages/grumpy/pythonparser/algorithm.py", line 41, in visit
    return self._visit_one(obj)
  File "/home/avdhesh/grumpy/build/lib/python2.7/site-packages/grumpy/pythonparser/algorithm.py", line 32, in _visit_one
    return getattr(self, visit_attr)(node)
  File "/home/avdhesh/grumpy/build/lib/python2.7/site-packages/grumpy/compiler/stmt.py", line 320, in visit_Module
    self._visit_each(node.body)
  File "/home/avdhesh/grumpy/build/lib/python2.7/site-packages/grumpy/compiler/stmt.py", line 734, in _visit_each
    self.visit(node)
  File "/home/avdhesh/grumpy/build/lib/python2.7/site-packages/grumpy/pythonparser/algorithm.py", line 41, in visit
    return self._visit_one(obj)
  File "/home/avdhesh/grumpy/build/lib/python2.7/site-packages/grumpy/pythonparser/algorithm.py", line 32, in _visit_one
    return getattr(self, visit_attr)(node)
  File "/home/avdhesh/grumpy/build/lib/python2.7/site-packages/grumpy/compiler/stmt.py", line 296, in visit_ImportFrom
    for imp in self.block.root.importer.visit(node):
  File "/home/avdhesh/grumpy/build/lib/python2.7/site-packages/grumpy/pythonparser/algorithm.py", line 41, in visit
    return self._visit_one(obj)
  File "/home/avdhesh/grumpy/build/lib/python2.7/site-packages/grumpy/pythonparser/algorithm.py", line 32, in _visit_one
    return getattr(self, visit_attr)(node)
  File "/home/avdhesh/grumpy/build/lib/python2.7/site-packages/grumpy/compiler/imputil.py", line 102, in visit_ImportFrom
    raise util.ImportError(node, msg)
grumpy.compiler.util.ImportError: line 1: wildcard member import is not implemented: from ExtractMsg import *

**Comment by aisk** [comment]
Yes this is a known issue if you take a look at the error message, or the wiki: https://github.com/google/grumpy/wiki/Missing-features#language-features

**Comment by trotterdylan** [comment]
There was some work done around this in https://github.com/google/grumpy/pull/263

The PR is a bit stale, but it's a good starting point for implementing wildcard imports.
